scipy/spatial/distance.py

```diff
@@ -282,6 +282,15 @@ def _validate_minkowski_kwargs(X, m, n, **kwargs):
     return kwargs
 
 
+def _validate_whamming_kwargs(X, m, n, **kwargs):
+    if 'w' not in kwargs:
+        kwargs['w'] = 1.
+
+    w = kwargs['w']
```

Comment:
I think the few lines above can be simplified with something like `w = kwargs.get('w', 1.)`